example/spec/syncGranule/SyncGranuleDuplicateHandlingErrorSpec.js

```diff
@@ -0,0 +1,60 @@
+const fs = require('fs');
+const { Collection } = require('@cumulus/api/models');
+const { addCollections, buildAndExecuteWorkflow } = require('@cumulus/integration-tests');
+
+const { loadConfig } = require('../helpers/testUtils');
+const config = loadConfig();
+
+describe('The Sync Granules workflow is configured to handle duplicates as an error', () => {
+  const inputPayloadFilename = './spec/syncGranule/SyncGranule.input.payload.json';
+  const inputPayload = JSON.parse(fs.readFileSync(inputPayloadFilename));
+  const collection = { name: 'MOD09GQ_duplicateHandlingError', version: '006' };
+  const provider = { id: 's3_provider' };
+
+  process.env.CollectionsTable = `${config.stackName}-CollectionsTable`;
+  const c = new Collection();
+
+  beforeAll(async () => {
+    const collectionsDirectory = './data/collections/syncGranule';
```

Comment:
@laurenfrederick I did actually decide to create new test collection data for these tests that I create/tear down, because I agree that ideally tests should manage their own data

Reply:
👍 I plan on doing this for the existing suites next.